### DISCUSSION TITLE
[FEATURE] Improve usability for non-mouse users

### DISCUSSION THREAD

**Discussion by tcmetzger** [bokeh/bokeh#11268] [General]
Currently, interactive elements of a Bokeh plot are not usable without a touch/pointing device (specifically the toolbar). At least that is what I found with Firefox and Chrome both on Win10 and Ubuntu (using https://docs.bokeh.org/en/latest/docs/user_guide/plotting.html#creating-figures for reference).

Improving usability for non-mouse users would benefit users with physical disabilities or users of assistive technologies such as screen readers, for example.

One relatively simple solution would be to include the Bokeh toolbar in the browser's standard tab order. If that proves to not be feasible (e.g. because of limitations of the canvas), we could consider assigning key bindings to each tool, potentially similar to what we are currently doing with the [edit tools](https://docs.bokeh.org/en/latest/docs/user_guide/tools.html#edit-tools). 

Some of the existing tools could also be updated to be more easily used with a keyboard, for example by enabling the use of arrow keys with the pan tool or enabling "+" and "-" with the wheel zoom tool.

**Comment by mattpap** [comment]
> (...) to include the Bokeh toolbar in the browser's standard tab order. If that proves to not be feasible (e.g. because of limitations of the canvas)

Toolbar is DOM/CSS so it can participate in tab order. It's just that its implementation is bad, using plain `<div>` as buttons.

**Comment by mattpap** [comment]
@tcmetzger, please split this issue into multiple feature requests, as this issue is not actionable, i.e. it's not clear when this issue would be closed. If you just want to discuss the broad usability issue, please convert this to a discussion ("Convert to discussion" in the side panel).

**Reply by tcmetzger** [reply]
You are right - here is the separat issue I created for the tab order: https://github.com/bokeh/bokeh/issues/11266

**Comment by bryevdv** [comment]
sidebar: I had not actually noticed "Convert to Discussion" before, I will probably make extensive use of that on the next full issue tracker triage I perform. I do agree this would be good as a discussion that can then be used to precipitate specific issues out of.

**Comment by tcmetzger** [comment]
@mattpap I'm glad to hear that the tab order would be possible, I'll make a separate issue for that. I also was not aware of the "convert do discussion" button, and I agree that this does make more sense as a discussion!

**Comment by tcmetzger** [comment]
@mattpap What are your thoughts on:

> Some of the existing tools could also be updated to be more easily used with a keyboard, for example by enabling the use of arrow keys with the pan tool or enabling "+" and "-" with the wheel zoom tool.

(I'd probably make that it's own, separate issue, even though it only makes sense in connction with #11266.)
